app/batching/ledger-lines/get-line-id.js

```diff
@@ -0,0 +1,9 @@
+const { ES, IMPS } = require('../../constants/schemes')
+
+const getLineId = (schemeId, lineId) => {
+  return [ES, IMPS].includes(schemeId) ? '' : lineId
```

Comment:
I'm not sure I completely understand what this code is doing. I know it's a conditional operator, just not entirely sure what the conditions are here

Reply:
If that makes any sense

Reply:
First step create an array which has the ES and IMPS scheme Ids in it.  So you'll have this `[9, 11]`

Then check does that array contain the scheme Id passed to the method with `includes` which will return `true` or `false`.

So if the scheme Id is ES or IMP then return an empty string as those submissions to DAX don't include line numbers.  If false then return the line id as is.

This way seemed more concise than.

```
if (schemeId === ES || schemeId === IMPS) {
  return ''
}
return lineId
```

Reply:
Oh okay, yeah that makes more sense now. Thanks :)